test/upgrade/prober/wathola/event/services.go

```diff
@@ -26,6 +26,12 @@ import (
 var mutex = sync.RWMutex{}
 var lastProgressReport = time.Now()
 
+const (
+	TagDuplicate = "DUPLICATE"
+	TagUnknown   = "UNKNOWN"
```

Comment:
Why we need this `UNKNOWN` type of error. We throw them, so it shouldn't be possible to UNKNOWN type of error be present, anywhere.

Reply:
I guess I should put UNCLASSIFIED instead, though it is not really used, but just as a fallback in case we don't want to keep coming up with new types.

Reply:
Let's remove it. It's confusing.

Reply:
We can remove the unknown tag/type, but I do think we can still keep a default throw in addition to DUPLICATE and MISSING to keep it flexible for future, eg. connection and system error etc.

Reply:
I removed UNKNOWN, but keep default so that we don't have to keep inventing new channels for every type, with the assumption that most of the time we just need to know there is an error.

Reply:
WDYT?